sqlglot/parser.py

```diff
@@ -1708,6 +1696,22 @@ def _parse_function(self):
         self._match_r_paren()
         return self._parse_window(this)
 
+    def _parse_user_defined_function(self):
+        this = self._parse_var()
+        if not self._match(TokenType.L_PAREN):
+            return this
+        expressions = self._parse_csv(lambda: self._parse_udf_kwarg(self._parse_id_var()))
+        self._match_r_paren()
+        return self.expression(exp.UserDefinedFunction, this=this, expressions=expressions)
+
+    def _parse_udf_kwarg(self, this):
```

Comment:
i don't think this needs to be passed this, just do parse_id_var inside here

then you can do self._parse_csv(self._parse_udf_kwarg)